packages/@ama-sdk/core/cli/files-pack.cts

```diff
@@ -67,18 +68,18 @@ void (async () => {
   };
 
   // Move files into the dist folder
-  const copies = files.map(async (glob) => {
+  const copies = files.map(async ({glob, cwdForCopy}) => {
     return watch ?
       import('chokidar')
-        .then((chokidar) => chokidar.watch(path.join(baseDir, glob)))
+        .then((chokidar) => chokidar.watch(path.posix.join(posixBaseDir, glob)))
```

Comment:
Actually `path.posix.join('c:\something', '*')` will result to `c:\something/*`. Not sure it is what you want

Reply:
Sorry did not see that before merging, but this has already been transformed to posix